src/test/librbd/managed_lock/test_mock_ReleaseRequest.cc

```diff
@@ -17,7 +19,36 @@ struct Traits<MockImageCtx> {
   typedef librbd::MockImageWatcher Watcher;
 };
 }
-}
+
+namespace managed_lock {
+
+template <>
+struct BreakLockRequest<librbd::MockImageCtx> {
```

Comment:
Not sure why these new template specializations are required -- ReleaseLock doesn't use them.